src/main/java/com/fontys/api/mutations/TeamMutation.java

```diff
@@ -0,0 +1,18 @@
+package com.fontys.api.mutations;
+
+import com.coxautodev.graphql.tools.GraphQLMutationResolver;
+import com.fontys.api.entities.Team;
+import com.fontys.api.service.TeamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TeamMutation implements GraphQLMutationResolver {
+
+    @Autowired
+    private TeamService teamService;
+
+    public Team createTeam(Long id, String name) {
+        return teamService.createTeam(id, name);
+    }
+}
```

Comment:
Don't know GraphQL, would like an explanation on what mutations are and why they're needed.